### ISSUE TITLE
Vimium doesnt work for app wunderlist.com

### ISSUE THREAD

**Issue by FantomX1** [philc/vimium#2826]

**Comment by smblott-github** [comment]
Do you mean their [Chrome app](https://chrome.google.com/webstore/detail/wunderlist-for-chrome/ojcflmmmcfpacggndoaaflkmcoblhnbh)?

If so, then likely this is because Chrome simply does not run extensions like Vimium on other extensions' pages.  If that's the case, then there is nothing which we can do about it.

**Comment by FantomX1** [comment]
No, I mean their website version. I realize that extension can't work on other extension. The navigation letters are displayed when pressing hotkey ctrl+F, bud list is not changed after "vimium-clicking" typing letters. Vimium is able at least focus the search bar there. But for example Vimperator from Firefox is able to correctly switch/navigate to specific todo list by pressing navigation numbers.

**Comment by cyberhck** [comment]
I don't understand the issue, I went to wunderlist.com, then pressing F gives me all hot keys, and pressing them does navigate me to the link, can you verify? also post which browser are you using, and which version of vimium you have :slightly_smiling_face:

**Comment by FantomX1** [comment]
Chrome Version 63.0.3239.84 (Official Build) (64-bit), Windows 10, Vimium version 1.62.4

**Comment by philc** [comment]
Closing; Wunderlist is now defunct.
